proto/serving.proto

```diff
@@ -11,6 +11,8 @@ package featureform.serving.proto;
 service Feature {
   rpc TrainingData(TrainingDataRequest) returns (stream TrainingDataRow) {}
   rpc FeatureServe(FeatureServeRequest) returns (FeatureRow) {}
+  rpc SourceData(SourceDataRequest) returns (stream SourceDataRow) {}
+  rpc SourceColumns(SourceColumnRequest) returns (SourceDataColumns) {}
```

Comment:
https://cloud.google.com/apis/design/naming_convention#method_names

It might be too late now but the convention is VerbNoun
rpc GetSourceColumns(GetSourceColumnRequest) returns ...

Reply:
Yeah, these serving protos definitely don't follow the conventions, whereas `metadata.proto` does. We should fix this.